packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-script-loader.ts

```diff
@@ -120,7 +122,7 @@ export default class PaypalCommerceScriptLoader {
             ...enableAPMsFunding,
         ];
 
-        return {
+        const basicConfig: PaypalCommerceScriptParams = {
```

Comment:
Out of curiosity why not do something like 
```
return {
    'client-id': clientId,
    /// rest properties
    ...(isDeveloperModeApplicable && { 'buyer-country': buyerCountry })
}
```

Reply:
I thought that its better separate dev config and basic one to a separate variables. I might be easier to understand which one for what